Mathlib/GroupTheory/Perm/Sign.lean

```diff
@@ -3,8 +3,13 @@ Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes
 -/
+
 import Mathlib.GroupTheory.Perm.Support
-import Mathlib.GroupTheory.OrderOfElement
+import Mathlib.Tactic.NormNum.Ineq
+import Mathlib.Algebra.CharZero.Lemmas
+import Mathlib.Logic.Equiv.Fin
+import Mathlib.Data.Finset.Sort
+import Mathlib.GroupTheory.Subgroup.Finite
 import Mathlib.Data.Finset.Fin
 import Mathlib.Data.Int.Order.Units
```

Comment:
```suggestion
import Mathlib.Algebra.CharZero.Lemmas
import Mathlib.Data.Finset.Fin
import Mathlib.Data.Finset.Sort
import Mathlib.Data.Int.Order.Units
import Mathlib.GroupTheory.Perm.Support
import Mathlib.GroupTheory.Subgroup.Finite
import Mathlib.Logic.Equiv.Fin
import Mathlib.Tactic.NormNum.Ineq
```

Reply:
You're adding an import?

Reply:
No, it's already there but Github doesn't let me do suggestions over deleted lines.